requirements-devel.txt

```diff
@@ -22,4 +22,6 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
--e git+https://github.com/mitsuhiko/flask.git#egg=Flask
+# FIXME: master branch in 0.13-dev release breaks the cli tests
+# -e git+https://github.com/mitsuhiko/flask.git#egg=Flask
```

Comment:
We will need to figure out why they break. If it's a bug in 0.13 then we need to report it, so that it can be fixed. If it's a change in 0.13, then it's better to fix it already because the release of 0.13 will come sooner or later.

Reply:
@lnielsen Ok should I investigate it more?